scripts/gui/thrive_gui.html

```diff
@@ -398,10 +398,18 @@
                 Chemoplast<hr><br> Cost: 40 mutation points<hr><br> 
                 Performs Process: Chemosynthesis<br>(1 Hydrogen Sulfide -> 2 Glucose)/Second<hr><br>
                 Allows for synthesis of glucose from hydrogen sulfide and atmospheric carbon dioxide.
-               <br>
-               Can be used to process the normally toxic soup containing hydrogen sulfide that comes out of hydrothermal vents at the bottom of the ocean. Into glucose.
-               </span>
+                <br>
+                Can be used to process the normally toxic soup containing hydrogen sulfide that comes out of hydrothermal vents at the bottom of the ocean. Into glucose.
+                </span>
                 <div id="ChemoplastIcon"></div>Chemoplast<br>40 MP</td>
+            <td id="addNucleus" class="OrganelleListItem">
+                <span class="tooltiptext">
+                Eukariote Nucleus<hr><br> Cost: 100 mutation points<hr><br>
+                Performs Process: Eukaryote Evolution<hr><br>
+                Allows for synthesis of RNA, MRNA, allow add chloroplasts and mitochondria
```

Comment:
I dont want to hAve to commit a tooltip change after we merge so can i suggest changing this line to:

Allows for synthesis of RNA, MRNA, allows cell to evolve membrane bound organelles.


Also change the displayed name in the tool tip and the button to just "Nucleus". Not  "Eukariote Nucleus"

Reply:
Also please make it scale the player cell down to 50% like the bacteria do right now. That is nessessary.

Reply:
Or we can make that a seperate issue i suppose

Reply:
The reduction is like is made in Microbe operations if is bacteria the species right, right?